static/js/components/Grid_test.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from "react"
+import { shallow } from "enzyme"
+import { assert } from "chai"
+
+import { Grid, Cell } from "./Grid"
+
+describe("Grid components", () => {
+  it("should render a grid with children", () => {
+    const wrapper = shallow(
+      <Grid>
+        <div className="child" />
+      </Grid>
+    )
+    assert.equal(wrapper.props().className, "mdc-layout-grid")
+    assert.ok(wrapper.find(".mdc-layout-grid__inner").exists())
+    assert.ok(wrapper.find(".child").exists())
+  })
+
+  //
+  ;[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].forEach(width => {
```

Comment:
Minor nitpick: could probably just test values `0` and `12` for `width` here, test passes no matter what `width` is: 0-12, or 99, or 'foo'.  It's just Flow that will complain about the bad values.

Reply:
yeah, I had added this mainly because the test file is checked with flow, but I can change it to just check 1 and 12